templates/scripts/sync-up-version.js

```diff
@@ -0,0 +1,42 @@
+const fs = require("fs");
+var path = require("path");
+const loadJsonFile = require("load-json-file");
+const writePkg = require("write-pkg");
+const semver = require("semver");
+
+const sdk_version = require(path.join(__dirname, "../../packages/sdk/package.json")).version;
+const sdk_name = require(path.join(__dirname, "../../packages/sdk/package.json")).name;
+
+function listFile(dir, list = []) {
+  var arr = fs.readdirSync(dir);
+  arr.forEach(function (item) {
+    var fullpath = path.join(dir, item);
+    var stats = fs.statSync(fullpath);
+    if (item === "node_modules") return list;
+    if (stats.isDirectory()) {
+      listFile(fullpath, list);
+    } else {
+      if (item === "package.json") list.push(fullpath);
+    }
+  });
+  return list;
+}
+
+const template_dir = path.join(__dirname, "..");
+const dep_pkgs = listFile(template_dir);
+for (file of dep_pkgs) {
+  let pkg_ = loadJsonFile.sync(file);
```

Comment:
usually, we use fs-extra and its `readJson`

Reply:
this load-json-file is leveraged by lerna to update json file, and writePkg as well, so it's better to keep this